zkevm-circuits/src/evm_circuit/execution/begin_tx.rs

```diff
@@ -599,4 +741,132 @@ mod test {
 
         assert_eq!(run_test_circuit_geth_data_default::<Fr>(block), Ok(()));
     }
+
+    #[test]
+    #[should_panic]
+    fn begin_tx_not_enable_skipping_invalid_tx_invalid_nonce() {
```

Comment:
The expected panic happens in bus_mapping or in circuit??
For circuit, we need to check all these 4 cases. I think we don't care if a user set the bus_mapping enable flag correctly, as long as the circuit keeps the bottomline safe.

|     tx     | real valid | real invalid | 
| ---------- | -------- | -------- |
| set valid    | positive | negetive|           
| set invalid |       n      |    p         | 

Please make sure your testcases covers all of them in the circuit.

Reply:
The expected panic happens in external-tracer. It's neither in bus_mapping nor in circuit.

Reply:
Added 4 testcases.